src/components/pages/account/SplitsHistogram.tsx

```diff
@@ -26,14 +26,10 @@ export default function SplitsHistogram({
 
   if (splits.length) {
     const interval = Interval.fromDateTimes(
-      splits[splits.length - 1].transaction.date,
-      splits[0]?.transaction.date,
+      splits[splits.length - 1].transaction.date.startOf('year'),
+      DateTime.now().endOf('year'),
```

Comment:
unrelated, fixes a bug where some accounts having data only in one year failed to load the histogram.